src/components/displayCats.jsx

```diff
@@ -0,0 +1,61 @@
+import {getReviewsByCat} from '../utils/api'
+import {useState, useEffect} from 'react'
+import { Link } from 'react-router-dom';
+
+
+const DisplayCats = ( { cats } ) => {
+    
+   
+
+    const [revData, setRevData] = useState([])
+
+    const cat = cats
```

Comment:
No need to redefine here. It means you are not tapped into the useState re-render.